spec/jobs/create_feedback_session_job_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe CreateFeedbackSessionJob, type: :job do
+  describe '#perform' do
+    let(:provider_user) { create(:user) }
+    let(:receiver_user) { create(:user) }
+    let(:feedback_session_date) { Time.zone.now.strftime("%Y-%m-%d") }
+    let(:tag_names) { ['Tag_1', 'Tag_2'] }
+
+    it "creates a feedback session" do
+      expect do
+        described_class.new.perform(
+          provider_user,
+          receiver_user,
+          feedback_session_date,
+          tag_names
+        )
```

Comment:
te recomiendo que antes del `it` definas un método perform para no tener que hacer esta llamada en cada test. Algo así:
```ruby
def perform
  described_class.perform(
          provider_user,
          receiver_user,
          feedback_session_date,
          tag_names
        )
end
``` 
Luego en el test haces 
```ruby
it "creates a feedback session" do
  expect { perform }.to change { FeedbackSession.count }.by(1)
end
```